sentry_sdk/scrubber.py

```diff
@@ -59,19 +59,36 @@
 
 
 class EventScrubber(object):
-    def __init__(self, denylist=None):
+    def __init__(self, denylist=None, recursive=False):
         # type: (Optional[List[str]]) -> None
```

Comment:
Missing the new arg in the type comment here:
```suggestion
        # type: (Optional[List[str]], bool) -> None
```